solara/lab/components/confirmation_dialog.py

```diff
@@ -0,0 +1,49 @@
+from typing import cast, Callable, List, Union, Optional
+
+import solara
+import reacton.ipyvuetify as v
+
+
+@solara.component
+def ConfirmationDialog(
+    is_open_rv: solara.Reactive[bool],
```

Comment:
Hm, `value` seems a bit too vague in my opinion... that makes sense for a switch etc, but maybe less so for a dialog... what is the "value" of a dialog?? It might be better to use something with "open". `is_open` seems to be more in line with what is usually used in Python, indicating a boolean. 

I'm not sure what the callback is for? Is it called when `is_open` (or whatever we end up calling it) changes value?